src/components/SearchPage/SearchPageComponent.js

```diff
@@ -18,6 +19,11 @@ export function SearchPageComponent() {
     1000,
   );
 
+  const handleClick = note => {
+    const data = { ...note, label: note.name };
+    dispatch(searchResultClickedAction({ note: data, edges: [] }));
```

Comment:
we have `note` as a function parameter, then we have `data` which is something different, and then we pass `{ note: data` as the action payload. So it's like we have two different notes here: `note` and `data`. This is confusing.